pkg/cri/config/config.go

```diff
@@ -353,6 +356,10 @@ type PluginConfig struct {
 	ImagePullProgressTimeout string `toml:"image_pull_progress_timeout" json:"imagePullProgressTimeout"`
 }
 
+type ImageDistributionHelper struct {
```

Comment:
We probably need exactly the same for pkg/cri/sbserver

Reply:
Thanks. It is also added to `pkc/cri/sbserver`